ci/scripts/github_commenter.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import re
+import logging
+from typing import Dict, Tuple, Any, Optional, List, Union
+
+from git_utils import GitHubRepo
+
+BOT_COMMENT_START = "<!---bot-comment-->"
```

Comment:
how does this related to bot-comment-foo-start?

Reply:
this marks the start of the whole comment, `the -foo-start` marks the boundaries of sections within the comment